tests/libcxx/Makefile

```diff
@@ -5,6 +5,10 @@ ifdef STRACE
 OPTS = --strace
 endif
 
+USER=$(shell id --user)
+GROUP=$(shell id --group)
+IDMAPPING = --enc-to-host-uid-map 0:$(USER) --enc-to-host-gid-map 0:$(GROUP)
```

Comment:
Likewise. Except for file system tests, which I see the possible need for ID mapping, I don't think ID mapping is required for other tests.

Reply:
hostfs and ext2 both read the host to determine what type it is. So yes there needs to be a mapping. we dont need the explicit everywhere now, but there does need to be a mapping even if it is default mapping